pattern-library/sass/patterns/_dropdown-menu.scss

```diff
@@ -0,0 +1,173 @@
+// ------------------------------
+// edX Pattern Library: Utilities - Dropdown Menu
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// #CONFIG
+// #UTILITIES
+// #GENERAL
+// #INDIVIDUAL CASES
+
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$dropdown-menu-button-font-size: font-size(small) !default;
+$dropdown-menu-list-font-size: font-size(base) !default;
+$dropdown-menu-z-index: 1001 !default;
+$dropdown-menu-button-text-align: left !default;
+$dropdown-menu-list-text-align: left !default;
```

Comment:
I agree that it's confusing, I will take a look into the UXPL code and see if there is a pattern already established. Thanks for calling this out, it will be good to agree on the approach for more code is written and confusion built into the UXPL.